docs/add-documentation.md

```diff
@@ -0,0 +1,27 @@
+# Add documentation
+
+The documentation in this repository describes the Thunderbird for Android project. It is intended to provide a comprehensive overview of the project, contribution, architecture, and the decisions made during development.
+
+We use [mdbook](https://rust-lang.github.io/mdBook/) to generate the documentation. The source files for the documentation are located in the `docs` directory. The `SUMMARY.md` file contains the table of contents for the documentation.
+
+## Building the Documentation
+
+To build the documentation, you need to have `mdbook` and it's dependencies installed. You can install `mdbook` using Cargo, the Rust package manager:
+
+```bash
+cargo install mdbook
+```
+
+Once you have `mdbook` installed, you can build the documentation by running the following command:
```

Comment:
The commands failed for me because I wasn't in the `docs/` directory. We should probably include a sentence mentioning that you need to change to that directory first.

Alternatively, we could move `book.toml` to the root directory and adjust its `book.src` property.

Reply:
Done